react-pool/public/assets/locales/de/translation.json

```diff
@@ -1,140 +1,141 @@
-{

```

Comment:
Why this file is being shown as completely changed? I had it updated from frank, is this a conflict?